recipes/django-leaflet/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "django-leaflet" %}
+{% set version = "0.22.0" %}
+{% set sha256 = "2a1f4f31bff2b3a58158372f7910053670483e3e11f8ac365ce75f3e3a4ac11c" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  noarch: python
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - django
+
+test:
+  commands:
+    - python -c "import django; from django.conf import settings; settings.configure(INSTALLED_APPS=['leaflet']); django.setup(); import leaflet;"
+
+about:
+  home: https://github.com/makinacorpus/django-leaflet
+  license: LGPL
```

Comment:
Once the feedstock is create you can update this with the license version: `LGPL 3.0` according to the text.